apps/metric_processors.go

```diff
@@ -51,6 +45,37 @@ func (p MetricsProcessorExcludeMetrics) Processors() []otel.Component {
 	)}
 }
 
+// globToRegex converts metrics glob patterns to regex patterns
+// For generating otel config, `$` needs to be escaped:
+// https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/metricstransformprocessor#rename-multiple-metrics-using-substitution
+func globToRegex(glob string, escapeForOtel bool) string {
+	dollarSign := "$"
```

Comment:
Why not do the `strings.Replace` in `Processors()` above, and eliminate the `escapeForOtel` parameter here?

Reply:
Done!